frontend/src/pages/Assistants/Assistants.tsx

```diff
@@ -4,82 +4,82 @@ import { useFetch } from '../../helpers/';
 import { Link } from 'react-router-dom';
 import img from '../assets/ronald.jpg';
 import {
-  AssistantItem,
-  CardBody,
-  ThumbNail,
-  ButtonContainer,
-  CardHeading,
-  Asst,
-  CheckList,
-  AssistantHeader,
-  HeaderWrapper,
+    AssistantItem,
+    CardBody,
+    ThumbNail,
+    ButtonContainer,
+    CardHeading,
+    Asst,
+    CheckList,
+    AssistantHeader,
+    HeaderWrapper,
 } from './Assistants.styling';
 import { Assistant } from './types';
 
 interface AssistantsEnum extends Array<Assistant> {}
 
 const AssistantCard = (assistant: Assistant) => {
-  return (
-    <>
-      <Link to={`/assistants/${assistant.ast_id}`}>
-        <AssistantItem data-testid='assistant-item'>
-          <ThumbNail src={img} alt={assistant.full_name} />
-          <CardBody>
-            <CardHeading>
-              <h1>{assistant.full_name}</h1>
-            </CardHeading>
-            <div
-              style={{
-                display: 'flex',
-                justifyContent: 'space-between',
-                width: '100%',
-              }}
-            >
-              <CheckList>
-                <h3>Checklist Items</h3>
-                <div>{assistant.itemCount}</div>
-              </CheckList>
-              <Asst>
-                <h3>Available Houses</h3>
-                <div>{assistant.houseCount}</div>
-              </Asst>
-              <ButtonContainer>
-                <Button
-                  text='House Availability'
-                  datatestid='assistant-button'
-                />
-              </ButtonContainer>
-            </div>
-          </CardBody>
-        </AssistantItem>
-      </Link>
-    </>
-  );
+    return (
+        <>

```

Comment:
Your prettier seems to work with tabs not spaces or the other way around

so you cause a lot of changes in every file you touch
Can you change that ?